docs/language/src/concepts/translations.md

```diff
@@ -46,8 +58,93 @@ export component MenuItem {
 
 ## Extracting the String from the files
 
-**TODO**
+We provide a `slint-tr-extractor` tool that parse .slint files and can generate a .po file.
+You can run it like so
+
+```sh
+find -name \*.slint | xargs slint-tr-extractor -o MY_PROJECT.pot
+```
+
+This will create a file called `MY_PROJECT.pot`
+(but you should replace MY_PROJECT with your actual project name, see bellow for naming)
 
 ## Doing the translations
 
-**TODO**
\ No newline at end of file
+`.pot` file are [Gettext](https://www.gnu.org/software/gettext/) template files. It is the same as a `.po` file, but doesn't contain actual
+translations. They can be edited by hand with a text editor, or there are a few tools you can use to translate them, option includes
+ - [poedit](https://poedit.net/)
+ - [OmegaT](https://omegat.org/)
+ - [Lokalize](https://userbase.kde.org/Lokalize)
+ - [Transifex](https://www.transifex.com/) (web interface)
```

Comment:
I think we should have an introduction at the top of the file that outlines _all_ of the steps, i.e.

1. Marking translatable strings with @tr.
2. Extracting the tr marked strings into .pot files.
3. Generating translations by using various editor choices to create .po files from .pot files.
4. Compiling .po files to .mo files.
5. Loading them at run-time with Slint APIs.

Right now all these steps are scattered. Some here, some further down. I think it makes sense to outline all the steps first, to provide the big picture, and then go into details in the sections.

```suggestion
`.pot` file are [Gettext](https://www.gnu.org/software/gettext/) template files. It is the same as a `.po` file, but doesn't contain actual
translations. They can be edited by hand with a text editor, or there are a few tools you can use to translate them, option include:
 - [poedit](https://poedit.net/)
 - [OmegaT](https://omegat.org/)
 - [Lokalize](https://userbase.kde.org/Lokalize)
 - [Transifex](https://www.transifex.com/) (web interface)
```

Reply:
> Right now all these steps are scattered. Some here, some further down. I think it makes sense to outline all the steps first, to provide the big picture, and then go into details in the sections.

What your describe is just the outline of the document.  (But maybe the syntax of @tr can be made sub-sections instead of sections)